src/Helpers/Exporting/Json/Decode.elm

```diff
@@ -103,6 +103,9 @@ tblTypeDecoder typ =
         "closed" ->
             closed
 
+        "S⁺" ->
+            unaryRule Tableau.Splus
```

Comment:
I’d rather call the rule “assumption” internally in the code and in the exports, and kept S+ as UI display name only.